content/_en/landing/create_an_account.md

```diff
@@ -36,5 +36,5 @@ steps:
 permalink: /create-an-account/
 one_account_banner: true
 twitter_card: large
-image: /assets/img/login-gov-600x314.png
```

Comment:
Is this image unused now? Can we remove it?

Reply:
@aduth I don't think that field is used anywhere. I will remove the field from Netlify as it isn't used and not being referenced anywhere.

Reply:
I was wondering about the image asset itself: https://github.com/18F/identity-site/blob/main/assets/img/login-gov-600x314.png

From what I can tell, this was the only place we were referencing it, so if it's unused, it seems to me that we could just remove it from the codebase?

Reply:
So after further review and investigation, I am going to keep this value. Here is why:

The `page.image` value was being used to populate the image meta tags in meta.html. If you check on line 43 in `meta.html`, you will see that if the `page.image` attribute is not present, it will fall back to the `site.image` attribute. [This is the image that page.url references](https://github.com/18F/identity-site/blob/main/assets/img/login-gov-600x314.png) while [this is the image that site.url references](https://github.com/18F/identity-site/blob/main/assets/img/login-gov-288x288.png)

As you will notice, while they are essentially the same image, the `site.image` version is cropped. I found in a [css-tricks article](https://css-tricks.com/essential-meta-tags-social-media/) that noted that URL of the image should be at least 600x315 for Facebook. The `page.url` value fits the requirement for Facebook (Twitter is less stringent on these rules).

After reading the article , I have questions that are outside the scope of this PR:

1. Should we update the image to include the missing height pixel per Facebook recommendations? 
2. Per the css-tricks article, the social image for Twitter should be"a URL to a unique image representing the content of the page. You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages" Should we consider another image for social platforms? 

I would like to know what the team thinks regarding the above questions.

Reply:
Good questions! Here's my take on them:

1. It seems like we might want to set the site-wide fallback to the wider version of the image, based on Facebook's recommended aspect ratio. As you noted, the 600x314 version we have isn't correct per the recommendations, so we might need to replace it with either a 600x315 or 1200x630 image. I found [additional guidance in Twitter's documentation](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image) which seems to indicate that an aspect ratio of 2:1 is preferred for Twitter images. We may want distinct images for `<meta property="og:image">` and `<meta property="twitter:image">` ? Or we could use their debugging tools to see if one image can work well enough for both (since 1.91:1 and 2:1 are pretty close).
2. I'm curious to know more about this uniqueness requirement. If I had to guess, I would imagine it's recommended to be unique so that Twitter readers would have a better understanding of what to expect the page to contain, but a generic fallback may not be strictly disqualifying? Regardless, I think we should probably want unique images where reasonable for as many pages as we can, like what you'd implemented in this pull request.

I could envision a few follow-on tasks:

- Set the default site image to a 600x315 (or 1200x630) generic image
   - Test [Twitter validator](https://cards-dev.twitter.com/validator) to confirm this is valid, or if we'll need a separate 2:1 image
   - Remove the then-unused 288x288 image
- Remove page-specific `image: /assets/img/login-gov-600x314.png` (since this is implicit, inherited by the new site default)
- Where possible, set page-specific `image:` to a unique image appropriate for the page

Do those sound agreeable to you?

Aside: I now see that I was mistaken in my original comment, thinking that we no longer referenced `login-gov-600x314.png` anywhere in the code. I see we do on several pages, as you point out. Sorry for the mix-up!